index.html

```diff
@@ -1,16 +1,410 @@
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal7.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Chocolate Cream</h6>
+                  <p>Chocolate and cream laced heaven</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal3.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Raspberry Delight</h6>
+                  <p>Cake with raspberry toppings. Hmmmm!</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal4.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Four Cupcakes</h6>
+                  <p>Four blue cupcakes for every ocassion</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal5.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Sliced Cake With Tea</h6>
+                  <p>Every Tired Workers Dream</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal6.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Vanilla Cake </h6>
+                  <p>Laced with Chocolate to taste</p>
+                </figcaption>
+              </figure>
+            </div>
+             </section>
+    </div>
+    <section id="contact" class="bg-secondary">
+      <div class="container">
+        <form class="py-5 text-white">
+          <h2>Sign up for Newsletter</h2>
+          <p>Receive the latest news and offers by signing up to our newsletter</p>
+          <div class="row">
+            <div class="col-12 col-sm-6">
+            <div class="form-group">
+              <label for="input-name" class="sr-only">Your Name:</label>
+              <input type="text" class="form-control" placeholder="Firstname Lastname" id="input-name">
+            </div></div>
```

Comment:
This is syntactically correct, but a little bit misleading. If it were up to me, I'd use a new line to close the div.